charts/alertmanager/values.yaml

```
@@ -57,3 +57,5 @@ customRoutes: []
 #   match_re:
 #     tier: platform
 #     namespace: ^(test-ns.*)
+
+enableNonRFC: false
```

Can we be more specific with this variable name?

```suggestion
enableNonRFC1918: false
```

---

changed it in my most recent commits.